app/scripts/main.js

```diff
@@ -40,6 +40,7 @@ import 'scripts/controllers/journalUploadReview.js';
 // Directives
 import 'scripts/directives/adminNav.js';
 import 'scripts/directives/autoFocus.js';
+import 'scripts/directives/faIcon.js';
```

Comment:
These imports are (mostly) sorted. Could you move this one after `datepicker.js`?